packages/fxa-shared/db/models/auth/session-token.ts

```diff
@@ -133,7 +133,7 @@ export class SessionToken extends BaseToken {
     tokenVerificationCode?: string;
     tokenVerificationCodeExpiresAt: number;
   }) {
-    return SessionToken.callProcedure(
+    return this.callProcedure(
```

Comment:
This is something that maybe should have been called out on the previous refactor, because it came up there as well. 

For some reason when binding Knex from the admin panel, calling static methods would result in runtime errors. Referencing 'this' fixes the problem. Part of me wants to fix this on all classes in fxa-shared, so they are consistent. Maybe a followup item should be filed for this issue.  with a good description of why the previous approach fails, and an example of what the transpiled typescript actually looks like.

Here's a distilled example of how Knex is bound in the auth-server:


```
export class DatabaseService implements OnModuleDestroy {

    public sessionTokens: typeof SessionToken;

    constructor(configService: ConfigService<AppConfig>) {

        // Results in a run time error. Static methods on SessionToken that are invoked 
        // from the context of this.sessionTokens will fail.
        this.sessionTokens = SessionToken.bindKnex(this.knex);
    }
}
```